script.js

```diff
@@ -14,3 +15,228 @@ navItems.forEach((navItem) => navItem.addEventListener('click', () => {
+                        <ul>
+                          <li>${cardsInfo[0].tech_stack.tech0}</li>
+                          <li>${cardsInfo[0].tech_stack.tech1}</li>
+                          <li>${cardsInfo[0].tech_stack.tech2}</li>
+                          <li>${cardsInfo[0].tech_stack.tech3}</li>
+                        </ul>
+                      </div>
+                      <button onclick='openfirst()' id="project-button" type="submit" href='#modal'>See Project</button>
+                    </div>`;
+
+const modalMobile = ` <div id="modal "class="modal">
+                        <header class='modal-header'>
+                          <h1 class="modal-heading font-size">${cardsInfo[1].title}</h1>
+                          <div class="close-btn" onclick="closeModal()">
+                            <div class="rotator1"></div>
+                            <div class="rotator2"></div>
+                          </div>
+                        </header>
+                        <div class="tech-box popup">
+                          <ul>
+                            <li>${cardsInfo[1].tech_stack.tech1}</li>
+                            <li>${cardsInfo[1].tech_stack.tech2}</li>
+                            <li>${cardsInfo[1].tech_stack.tech3}</li>
+                          </ul>
+                        </div>
+                        <div class="content">
+                          <div id="img10"></div>
+                          <div class="right-section">
+                            <p class="description popup">${cardsInfo[0].details}</p>
+                            <div class="button-container">
+                              <button type="submit" id="seeLive" class="project-button popup">See Live<img src="icons/seeLive.svg" /></button>
+                              <button type="submit" id="seeSource" class="project-button popup">See Source<img src="icons/seeSource.svg" /></button>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                      <div class="overlay"></div>`;
+
+const modalfirst = `<div id="modal "class="modal">
+                      <header class='modal-header'>
+                        <h1 class="modal-heading">${cardsInfo[0].title2}</h1>
+                        <div class="close-btn" onclick="closeModal()">
+                          <div class="rotator1"></div>
+                          <div class="rotator2"></div>
+                        </div>
+                      </header>
+                      <div class="tech-box popup">
+                        <ul>
+                          <li>${cardsInfo[0].tech_stack.tech0}</li>
+                          <li>${cardsInfo[0].tech_stack.tech1}</li>
+                          <li>${cardsInfo[0].tech_stack.tech2}</li>
+                          <li>${cardsInfo[0].tech_stack.tech3}</li>
+                        </ul>
+                      </div>
```

Comment:
- [ ] As specified in the project requirements, Kindly make sure When the user clicks (or taps) the button to check project details, `the popup with details about the project appears`. As you may notice your modals are showing all the same exact data except the titles, whereas even the `details`, `pictures` should suppose to match with the project the user clicks at. Kindly consider changing that.